component/libfuse/libfuse.go

```diff
@@ -204,6 +206,20 @@ func (lf *Libfuse) Validate(opt *LibfuseOptions) error {
 		lf.negativeTimeout = defaultNegativeEntryExpiration
 	}
 
+	if lf.allowOther {
+		// Validate allow_other is enabled in fuse.conf file or not
+		cmd := exec.Command("bash", "-c", "cat /etc/fuse.conf | grep -i user_allow_other$")
+
+		var errb bytes.Buffer
+		cmd.Stderr = &errb
+		cliOut, err := cmd.Output()
+		data := string(cliOut)
+		if err == nil && data[0] == '#' {
```

Comment:
Add case for when `user_allow_other` is not present in /etc/fuse.conf

Reply:
done